ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.WebAPI/Controllers/ProjectsController.cs

```diff
@@ -38,5 +38,12 @@ public async Task<ActionResult<ProjectDTO>> CreateAsync([FromBody] CreateProject
             var projectDto = await _service.CreateAsync(project);
             return Created(string.Empty, projectDto);
         }
+
+        [HttpGet("Name")]
```

Comment:
use lowercase, and name it `names` because you return list of strings not just one name